app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinContract.kt

```diff
@@ -14,9 +14,14 @@ interface NewCreatePinContract {
         fun vibrate(duration: Long)
         fun showLoading(isLoading: Boolean)
         fun navigateBack()
+        fun navigateToBiometrics(pinCode: String)
+        fun navigateToMain()
     }
 
     interface Presenter : MvpPresenter<View> {
+        var pinMode: PinMode
```

Comment:
Why don't you just create a global value inside presenter and leave here a set method? 
Like `setMode` or smth.
We don't have variables in presenters

Reply:
Property style is better as for me, but let's make it consistent, okay